airbyte-integrations/connectors/destination-oracle/src/main/java/io/airbyte/integrations/destination/oracle/OracleDestination.java

```diff
@@ -103,6 +105,9 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     if (config.has("password")) {
       configBuilder.put("password", config.get("password").asText());
     }
+    if (config.has(JDBC_URL_PARAMS_KEY)) {
```

Comment:
Do we need check if JDBC_URL_PARAMS_KEY **isEmpty()** here?

Reply:
Have seen empty string support for other connectors (MySQL/MSSQL), that is why not added this condition.